data/sql/updates/pending_db_world/rev_1620842884434937800.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620842884434937800');
+
+DELETE FROM `creature_loot_template` WHERE `Item` = 7146;
```

Comment:
> Remove The Scarlet Key (ID 7146) from Scarlet Diviner, Scarlet Chaplain and Scarlet Monk loot tables (IDs 4291,4299,4540 respectively)

you are missing the creature clause in your `DELETE` statement. According to your comment, it should be something like;

```sql
DELETE FROM `creature_loot_template` WHERE `Item` = 7146 AND `Entry` IN (4291,4299,4540);
```

Reply:
Thanks, I'll add that :] I didn't put that in at first because those are the only entries that have `'Item'= 7146`, so I thought it would have the same effect.